editor/editor_inspector.cpp

```diff
@@ -901,6 +913,8 @@ void EditorProperty::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("set_label", "text"), &EditorProperty::set_label);
 	ClassDB::bind_method(D_METHOD("get_label"), &EditorProperty::get_label);
 
+	ClassDB::bind_method(D_METHOD("_set_read_only", "read_only"), &EditorProperty::_set_read_only);
+	ClassDB::bind_method(D_METHOD("set_global_read_only", "read_only"), &EditorProperty::set_global_read_only);
```

Comment:
This method is meant to be used only by the inspector, so I'd remove the binding.